push_story.php

```diff
@@ -162,14 +162,14 @@ function nprstory_push_meta_keys( $post_type = 'post' ) {
 		ON $wpdb->posts.ID = $wpdb->postmeta.post_id
 		WHERE $wpdb->posts.post_type = '%s'
 		AND $wpdb->postmeta.meta_key != ''
-		AND $wpdb->postmeta.meta_key NOT RegExp '(^[_0-9]wp_.+$)'
+		AND $wpdb->postmeta.meta_key NOT LIKE '_oembed_%'
 		AND $wpdb->postmeta.meta_key NOT RegExp '(^[0-9]+$)'
 	";
 	//AND $wpdb->postmeta.meta_key NOT RegExp '(^[_0-9].+$)'
 	$keys = $wpdb->get_col( $wpdb->prepare( $query, $post_type ) );
 	if ( $keys ) natcasesort( $keys );
 
-	//set_transient('ds_npr_' .  $post_type .'_meta_keys', $keys, 60*60*24); # 1 Day Expiration
+	set_transient( 'ds_npr_' . $post_type . '_meta_keys', $keys, 60*60*24 ); # 1 Day Expiration
```

Comment:
@jwcounts do you know why this transient setting was disabled/commented out previously?  Are there consequences to re-enabling?

Reply:
As far as I can tell, we're the first people to touch this file in about 4 years. I'm assuming that it was commented out because transients were probably newer and less reliable.

As for consequences, I'm assuming that setting a transient without any kind of caching mechanism installed probably doesn't do anything, and getting a transient returns a blank (which is accepted behavior for an expired transient too).

Reply:
I was also glad to take a stab at that MySQL query, because those dropdowns were returning over 10k entries apiece. It took the page a good 45 seconds to load each time, and most of them were _oembed_ transients.